docs/core/phone_home.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Phone Home
+nav_order: 15
+parent: Core
+---
+
+# Phone Home
+
+We're committed to being more aware of our products, how they are used and how they behave in different conditions and hardware.
+
+To do so, we actually need to expand how the Phone Home worked in NS6-NS7 by adding additional telemetry to be sent and processed for analysis.
```

Comment:
Could you add a command (e.g. `api-cli run get-facts`) to see a preview of what data is sent?

Can we state that data is always collected in anonymous form?

Reply:
> Could you add a command (e.g. api-cli run get-facts) to see a preview of what data is sent?

Was thinking the same thing once the PR went live :smile:, will add the line!

> Can we state that data is always collected in anonymous form?

As for now, yes, no info is actually fetched aside from `/etc/os-release`. We can always delete that statement.